src/__tests__/is-of-type.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { isOfType } from '../is-of-type'
+import { createActionCreator } from '../create-action-creator'
+
+describe('isOfType', () => {
+  const increment = createActionCreator('[Counter] increment')
+  const decrement = createActionCreator('[Counter] decrement')
+  const reset = createActionCreator(
+    '[Counter] reset',
+    resolve => (value: number) => resolve(value)
+  )
+
+  it('should work with single action type arg', () => {
+    expect(isOfType(increment.type, increment())).toBeTruthy()
+    expect(isOfType(decrement.type, increment())).toBeFalsy()
+    expect(isOfType(reset.type, reset(0))).toBeTruthy()
+  })
+
+  it('should work with multiple action type args', () => {
+    expect(
+      isOfType([increment.type, decrement.type, reset.type], increment())
+    ).toBeTruthy()
+    expect(isOfType([increment.type, reset.type], decrement())).toBeFalsy()
+    expect(isOfType([decrement.type, reset.type], reset(0))).toBeTruthy()
+  })
+
+  it('should work with single action arg', () => {
+    expect(isOfType(increment(), increment())).toBeTruthy()
+    expect(isOfType(decrement(), increment())).toBeFalsy()
+    expect(isOfType(reset(0), reset(1))).toBeTruthy()
```

Comment:
The body of the tests is similar. So we can use [`it.each`](https://jestjs.io/docs/en/api#2--describeeachtablename-fn-timeout-) template literal tag to make tests more readable. As a reference, you can check create-action.spec.ts.

https://github.com/thebrodmann/deox/blob/8b0bbcec09703f93eb4c89398b24d1a0a4117bc8/src/__tests__/create-action.spec.ts#L4-L23

Reply:
completed! https://github.com/thebrodmann/deox/pull/127/commits/d054a760698a4998968c38eb7d85058d0d91abe8
I want you to review again when you have time 🙏